packages/react-components/react-positioning/src/usePositioning.ts

```diff
@@ -300,90 +45,63 @@ export function usePositioning(
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   arrowRef: React.MutableRefObject<any>;
 } {
+  const { targetDocument } = useFluent();
   const { enabled = true } = options;
-  const isFirstMount = useFirstMount();
-
-  const popperOriginalPositionRef = React.useRef<string>('absolute');
-  const resolvePopperOptions = usePopperOptions(options, popperOriginalPositionRef);
-
-  const popperInstanceRef = React.useRef<PopperInstance | null>(null);
-
-  const handlePopperUpdate = useEventCallback(() => {
-    popperInstanceRef.current?.destroy();
-    popperInstanceRef.current = null;
+  const resolvePositioningOptions = usePositioningOptions(options);
 
+  const forceUpdate = useEventCallback(() => {
     const target = overrideTargetRef.current ?? targetRef.current;
-
-    let popperInstance: PopperInstance | null = null;
-
-    if (canUseDOM() && enabled) {
-      if (target && containerRef.current) {
-        popperInstance = PopperJs.createPopper(
-          target,
-          containerRef.current,
-          resolvePopperOptions(target, containerRef.current, arrowRef.current),
-        );
-      }
+    if (!canUseDOM || !enabled || !target || !containerRef.current) {
+      return;
     }
 
-    if (popperInstance) {
-      /**
-       * The patch updates `.forceUpdate()` Popper function which restores the original position before the first
-       * forceUpdate() call. See also "positionStyleFix" modifier in usePopperOptions().
-       */
-      const originalForceUpdate = popperInstance.forceUpdate;
-
-      popperInstance.isFirstRun = true;
-      popperInstance.forceUpdate = () => {
-        if (popperInstance?.isFirstRun) {
-          popperInstance.state.elements.popper.style.position = popperOriginalPositionRef.current;
-          popperInstance.isFirstRun = false;
+    const { placement, middleware, strategy } = resolvePositioningOptions(
+      target,
+      containerRef.current,
+      arrowRef.current,
+    );
+
+    // Container is always initialized with `position: fixed` to avoid scroll jumps
+    // Before computing the positioned coordinates, revert the container to the deisred positioning strategy
+    Object.assign(containerRef.current.style, { position: strategy });
+    computePosition(target, containerRef.current, { placement, middleware, strategy })
+      .then(({ x, y, middlewareData, placement: computedPlacement }) => {
+        writeArrowUpdates({ arrow: arrowRef.current, middlewareData });
+        writeContainerUpdates({
+          container: containerRef.current,
+          middlewareData,
+          placement: computedPlacement,
+          coordinates: { x, y },
+          lowPPI: (targetDocument?.defaultView?.devicePixelRatio || 1) <= 1,
+          strategy,
+        });
+      })
+      .catch(err => {
+        // This is super duper stuuuuuuupid
+        // FIXME for node > 14
```

Comment:
lets create issue for this and link that here so we can track it / fix it

Reply:
see https://github.com/microsoft/fluentui/pull/24254#discussion_r940160910 I don't think we can fix it tbh